pavelib/servers.py

```diff
@@ -28,9 +28,10 @@ def run_server(system, settings=None, port=None, skip_assets=False, contracts=Fa
     if not settings:
         settings = DEFAULT_SETTINGS
 
-    if not skip_assets:
-        # Local dev settings use staticfiles to serve assets, so we can skip the collecstatic step
-        args = [system, '--settings={}'.format(settings), '--skip-collect', '--watch']
+    if asset_settings:
+        args = [system, '--settings={}'.format(asset_settings), '--skip-collect', '--watch']
```

Comment:
I'm confused about "--skip-collect", which appears on lines 32 and 34. You might need to explain this to me (if this code is actually correct).

Is it necessary to have both collect_static and asset_settings?